pkgs/applications/misc/parsec/bin.nix

```diff
@@ -0,0 +1,72 @@
+{ stdenvNoCC, stdenv
+, lib
+, dpkg, autoPatchelfHook, makeWrapper
+, fetchurl
+, alsaLib, openssl, udev
+, libglvnd
+, libX11, libXcursor, libXi, libXrandr
+, libpulseaudio
+, libva
+, ffmpeg
+}:
+
+stdenvNoCC.mkDerivation {
+  pname = "parsec-bin";
+  version = "150_28";
+
+  src = fetchurl {
+    url = "https://builds.parsecgaming.com/package/parsec-linux.deb";
```

Comment:
This is a no. We need a proper versioned URL that it is stable (always point to the same artefact). If this is not possible please archive this somewhere and link it to the archive. You can even keep both the archive and the upstream links by using `urls` instead of `url` here.

Reply:
Here is the latest [archive.org](https://web.archive.org/web/20220701000000*/https://builds.parsecgaming.com/package/parsec-linux.deb) link that I could found (from July 22th): https://web.archive.org/web/20220622215230id_/https://builds.parsecgaming.com/package/parsec-linux.deb

Reply:
That is actually still the latest version (150_28). The program downloads updates to `parsecd-$VERSION.so` on its own, so the installer download itself has been stable for some time.